lib/src/constants/constants.dart

```diff
@@ -4,4 +4,7 @@ class CategoriesScreen {}
 
 class CryptoCurrencyScreen {}
 
-class CryptoDetailScreen {}
\ No newline at end of file
+class CryptoDetailScreen {
+  static const String symbolImage = 'assets/images/bitcoin_symbol.jpg';
```

Comment:
Let's rename this to something like `symbolImagePath` for clarity

Reply:
I fixed it.